zhaquirks/xiaomi/aqara/switch_aq2.py

```diff
@@ -93,8 +95,20 @@ class SwitchAQ2(XiaomiCustomDevice):
         },
         (TRIPLE_PRESS, TRIPLE_PRESS): {
             COMMAND: COMMAND_TRIPLE,
-            CLUSTER_ID: 0,
+            CLUSTER_ID: 6,
             ENDPOINT_ID: 1,
             ARGS: {ATTRIBUTE_ID: 32768, ATTRIBUTE_NAME: UNKNOWN, VALUE: 3},
         },
+        (QUADRUPLE_PRESS, QUADRUPLE_PRESS): {
+            COMMAND: COMMAND_ATTRIBUTE_UPDATED,
+            CLUSTER_ID: 6,
+            ENDPOINT_ID: 1,
+            ARGS: {ATTRIBUTE_ID: 32768, ATTRIBUTE_NAME: UNKNOWN, VALUE: 4},
+        },
+        (QUINTUPLE_PRESS, QUINTUPLE_PRESS): {
```

Comment:
this can also be emitted by pressing the button on the device. Please remove it from the click actions as it isn't guaranteed to be a 5x press that causes the event.